src/js/index.js

```diff
@@ -835,6 +835,10 @@ export const hpe = deepFreeze({
       },
     },
     weight: 700,
+    extend: ({ theme }) =>
```

Comment:
darn, I was hoping we could do this via something like `theme.heading.color`, similar to how we have `theme.heading.weight`.

Reply:
I think that would be a good enhancement. Grommet doesn't currently support that, so I was going with what functionality already existed, but I can file a PR on grommet for this.

Reply:
I've incorporated the new grommet changes.